src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar-list calendar-list--month-length-28 calendar-list--month-length-29 calendar-list--month-length-30 calendar-list--month-length-31">
+      <li class="calendar-list__item calendar-list__item--1 calendar-list__item--start-day-mon calendar-list__item--start-day-tue calendar-list__item--start-day-wed calendar-list__item--start-day-thu calendar-list__item--start-day-fri calendar-list__item--start-day-sat calendar-list__item--start-day-sun"></li>
```

Comment:
`calendar-list__item--start-day` modifier should be also one and it should be on the calendar element, not day